src/parser/postgresparser.cpp

```diff
@@ -528,7 +528,11 @@ std::unique_ptr<AbstractExpression> PostgresParser::ColumnRefTransform(ParseResu
         table_name = "";
       } else {
         auto next_node = reinterpret_cast<Node *>(fields->head->next->data.ptr_value);
-        col_name = reinterpret_cast<value *>(next_node)->val_.str_;
+        if (next_node->type == T_A_Star) {
+          col_name = "*";
```

Comment:
Should this maybe generate a star expression instead? @tpan496

Reply:
I can't directly return `StarExpression` in this case, because there is a table qualifier. We only want all the columns for the given table.

Reply:
I agree with @lmwnshn. I don't think we should hardcode `col_name = "*"` here. That seems like a hack.